### ISSUE TITLE
Wheres the website link?

### ISSUE THREAD

**Issue by Glixh** [Drew690/Mokeleven#1]
Ive been looking for the link to the website but I can't find it can you help me?

**Comment by Drew690** [comment]
yeah you just have to go to actions then click one of them and link shoud be there
